buf.gen.yaml

```diff
@@ -7,14 +7,14 @@ managed:
       - buf.build/googleapis/googleapis
       - buf.build/grpc-ecosystem/grpc-gateway
 plugins:
-  - remote: buf.build/protocolbuffers/plugins/go:v1.28.0-1
+  - plugin: buf.build/protocolbuffers/go:v1.30.0
```

Comment:
Newer versions of the `buf` CLI use the `plugin` key for both local and remote plugin references.